matlab/test/arrow/tabular/tTabularDisplay.m

```diff
@@ -124,6 +153,167 @@ function ManyRowsAndColumns(testCase, TabularType)
+            expectedDisplayOfData = [newline, ...
+                                     '    Schema:', newline, ...
+                                     newline ...
+                                     '        ', var1Field, ' | ', var2Field, ' | ', var3Field, newline, ...
+                                     newline, ...
+                                     '    First Row:', newline, ...
+                                     newline, ...
+                                     '        ', '"a"', ' | ', '"', '<a href="http://www.foo.com">foo</a>', '"', ' | ', 'null', newline, ...
+                                     newline];
+
+            % hotlinks is turned off
+            actualDisplay = evalc('feature(''hotlinks'',''off'');disp(arrowTabularObj)');
+            testCase.verifySubstring(actualDisplay, expectedDisplayOfData);
+
+            % hotlinks is turned on
+            actualDisplay = evalc('feature(''hotlinks'',''on'');disp(arrowTabularObj)');
+            testCase.verifySubstring(actualDisplay, expectedDisplayOfData);
+        end
+
+        function displayClassNameWhenDataIsNotArray(testCase, TabularType)
+            % When the datatype of a column is not an homogeneous array
+            % type, the class name instead of the actual data will be
+            % displayed.
+            import arrow.internal.test.display.makeLinkString
+
+            t = table(datetime(2023,1,[1;2;3]),table([1;2;3],[4;5;6]),seconds([1;2;3]));
+            arrowTabularObj = TabularType.FromTableFcn(t);
+            actualDisplay = evalc('disp(arrowTabularObj)');
+
+            classNameString = makeLinkString(FullClassName=TabularType.FullClassName, ...
+                ClassName=TabularType.ClassName, BoldFont=true);
+            header = compose("  Arrow %s with %s rows and %s columns:", classNameString, getNumString(3), getNumString(3));
+            var1Field = char(makeFieldString("Var1", "Timestamp" , "arrow.type.TimestampType"));
+            var2Field = char(makeFieldString("Var2", "Struct" , "arrow.type.StructType"));
+            var3Field = char(makeFieldString("Var3", "Time64", "arrow.type.Time64Type"));
+            expectedDisplay = [char(header), newline, ...
+                               newline, ...
+                               '    Schema:', newline, ...
+                               newline, ...
+                               '        ', var1Field, ' | ', var2Field, ' | ', var3Field, newline, ...
+                               newline, ...
+                               '    First Row:', newline, ...
+                               newline, ...
+                               '        ', '2023-01-01 00:00:00.000000 | <Struct> | 00:00:01.000000', newline,...
+                               newline];
+
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+        end
+
+        function displayInvalidData(testCase, TabularType)
+            % Display "null" for invalid value
```

Comment:
For consistency with the other test cases, could you start the test case comments with something like "Verify that..."